### ISSUE TITLE
CMake 3.11.0 - Deprecation Warning

### ISSUE THREAD

**Issue by Gargaj** [glfw/glfw#1244]
https://github.com/glfw/glfw/blob/master/CMakeLists.txt#L10 is now causing a problem with latest CMake.

```
cmake : CMake Deprecation Warning at external/glfw/CMakeLists.txt:10 (cmake_policy):
At line:1 char:1
+ cmake -DBONZOMATIC_64BIT="$env:x64" -DBONZOMATIC_WINDOWS_FLAVOR:STRIN ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (CMake Deprecati...(cmake_policy)::String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
  The OLD behavior for policy CMP0042 will be removed from a future version
  of CMake.
  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
```

**Comment by elmindreda** [comment]
This should be fixed now with 474fa73e6cd846e19cfc71923c1422d51f1ea4a1. Thank you!

**Comment by ChenZon** [comment]
how can I do it
